ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -268,7 +289,9 @@
       ;; Add routes to create, update, delete, & validate concepts
       ingest-routes
 
+      ;; add routes to create, update, read, and delete generic concepts
       generic-document-routes
+      generic-document-routes-auxiliary
```

Comment:
so your suggesting we support both for a time?

Reply:
Just until we and Ryan agree on what the URL will be.